[Feature Request] Accept 1-tuples in `vale.Is`

I'm working on adding beartype support to `optype.numpy`, and wrote this:

```py
if not TYPE_CHECKING:
    try:
        import beartype.vale
    except ImportError:
        pass
    else:
        ToFloat64 = Annotated[
            ToFloat64,
            beartype.vale.Is[
                lambda x: (
                    isinstance(x, (float, int))
                    or (isinstance(x, np.generic) and np.can_cast(x, np.float64))
                ),  # This is Colin. Colin is a trailing comma. Colin is cool.
            ],
        ]
```

But the bear doesn't seem to like Colin:


```pytb
optype/numpy/__init__.py:50: in <module>
    from . import (
optype/numpy/_to.py:356: in <module>
    beartype.vale.Is[
.venv/lib/python3.14/site-packages/beartype/vale/_is/_valeis.py:304: in __getitem__
    self._die_unless_getitem_args_1(is_valid)
.venv/lib/python3.14/site-packages/beartype/vale/_is/_valeisabc.py:144: in _die_unless_getitem_args_1
    raise BeartypeValeSubscriptionException(
E   beartype.roar.BeartypeValeSubscriptionException: Beartype validator factory "Is" subscripted by two or more arguments (<function <lambda> at 0x72385b537060>,).
```

It says that `Is` is "subscripted by two or more arguments". But Colin isn't an argument; Colin is a comma, and Colin is cool.

So could cool comma Colin also be accepted by `beartype.vale.Is`?

--- a/beartype/vale/_is/_valeis.py
+++ b/beartype/vale/_is/_valeis.py
@@ -299,6 +299,10 @@ class _IsFactory(_BeartypeValidatorFactoryABC):
         '''
 
         # ..................{ VALIDATE                       }..................
+        # Unwrap a 1-tuple subscription (e.g., "Is[func,]") into its sole item.
+        if isinstance(is_valid, tuple) and len(is_valid) == 1:
+            is_valid = is_valid[0]
+
         # If this class was subscripted by either no arguments *OR* two or more
         # arguments, raise an exception.
         self._die_unless_getitem_args_1(is_valid)

--- a/beartype_test/a00_unit/a30_api/vale/_is/test_valeis.py
+++ b/beartype_test/a00_unit/a30_api/vale/_is/test_valeis.py
@@ -94,6 +94,17 @@ def test_api_vale_is_pass() -> None:
     IsExclamatory = Is[_is_exclamatory]
     IsFalseFake = Is[_is_false_fake]
 
+    # Validator produced by subscripting this factory with a 1-tuple, which is
+    # equivalent to subscripting with the sole item of that tuple directly.
+    IsLengthy1Tuple = Is[
+        lambda text: "dark" in text,
+    ]
+    assert isinstance(IsLengthy1Tuple, BeartypeValidator)
+    assert IsLengthy1Tuple.is_valid(
+        'Whose woods these are I think') is False
+    assert IsLengthy1Tuple.is_valid(
+        'The woods are lovely, dark and deep,') is True
+
     # Validator synthesized from the above validators with the domain-specific
     # language (DSL) supported by those validators.
     IsLengthyOrUnquotedSentence = IsLengthy | (IsSentence & ~IsQuoted)

